### DISCUSSION TITLE
[A-L] (2023/24) Foglio 16 - Esercizio 4

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#293] [Esercizi - Viaggi/Piazza]
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/edff44ac-3bd8-472c-9e79-8a31a4fe47f2)

**Comment by Elia-Belli** [comment]
Calcoliamo l'immagine degli elementi della base $\mathcal{E}$ tramite $T$:
$$T(1)=0=0\cdot 1+0\cdot t+0\cdot t^2$$

$$T(t)=1=1\cdot 1+0\cdot t +0\cdot t^2$$

$$T(t^2)=2t=0\cdot 1+2\cdot t +0\cdot t^2$$

La colonna $j$ della matrice $A$ associata a $T$ sono le coordinate in base $\mathcal{E}$ di $T(1),T(t),T(t^2)$:

$$A=\begin{bmatrix}0&1&0\\
0&0&2\\
0&0&0\end{bmatrix}$$

Siccome $A$ è singolare $\Rightarrow det(A)=0$

**Comment by CasuFrost** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/97389762/33f7c331-4289-4893-bd5c-b24692c7f381)

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/cd87cd11-ccd2-4e7d-98a8-6941804af633)
